vigetlabs/blendid
# http2 upgrade

## big picture
this adds a command to run after `yarn run blendid -- init` so that developers can take advantage of HTTP/2 multiplexing

this only affects the html, javascript and stylesheets directories within src and currently only works as expected with the default init task, though will most likely work with Drupal and Craft init tasks as well.

## to test it yourself
1. pull this branch down
2. run the standard `yarn install` command
3. copy the `extras/http2` directory and paste it into `node_modules/blendid/extras`
4. copy the `gulpfile.js/tasks/http2-upgrade.js` file and paste it into `node_modules/blendid/`gulpfile.js/tasks`
5. run `yarn run blendid -- init`
6. run `yarn run blendid -- http2-upgrade`
7. run `yarn run blendid` and you should see this page:

<img width="1500" alt="screen shot 2017-09-13 at 4 08 48 pm" src="https://user-images.githubusercontent.com/3211091/30403253-023a2808-989e-11e7-84c7-13acb307de58.png">


## todo before merging
- write a blog post to link to in the launch page
- ensure the HTTP/2 README links to the correct place
- remove blendid package from package.json
